### PR TITLE
fix: [CDS-85765]: add missing info about basic deployment using winrm

### PR DESCRIPTION
**Pull request by krishi0408**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screenshot.

### CODE REVIEW

File: docs/continuous-delivery/deploy-srv-diff-platforms/traditional/win-rm-tutorial.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Supported security protocols for WinRM include Kerberos and Windows New Technolo
 * Target host — In this guide, we use an AWS EC2 instance as the target host.
 * Credentials and other details for the EC2 Instance 
   * To configure and authenticate your WinRM credentials by using NTLM, you should have the domain name, username and password for the EC2 instance.
-
+* You should have [IIS service](https://learn.microsoft.com/en-us/iis/application-frameworks/scenario-build-an-aspnet-website-on-iis/configuring-step-1-install-iis-and-asp-net-modules#to-install-iis-and-aspnet-modules-on-windows-server-2012-using-the-ui) as well as [IIS website](https://learn.microsoft.com/en-us/iis/get-started/getting-started-with-iis/create-a-web-site)  installed in your machine.
```
Comment (acasian-harness) [inline docs/continuous-delivery/deploy-srv-diff-platforms/traditional/win-rm-tutorial.md:36]: we need to mention that in order to use Harness provided execution strategies, IIS service should be installed.

Comment (roshnisarangadharan) [reply docs/continuous-delivery/deploy-srv-diff-platforms/traditional/win-rm-tutorial.md:36]: install IIS website? Is it technically correct?

Comment (roshnisarangadharan) [reply docs/continuous-delivery/deploy-srv-diff-platforms/traditional/win-rm-tutorial.md:36]:
```suggestion
* To use Harness execution strategies, make sure you should have [IIS service](https://learn.microsoft.com/en-us/iis/application-frameworks/scenario-build-an-aspnet-website-on-iis/configuring-step-1-install-iis-and-asp-net-modules#to-install-iis-and-aspnet-modules-on-windows-server-2012-using-the-ui) installed on your machine. Fore more information, go to the [IIS website](https://learn.microsoft.com/en-us/iis/get-started/getting-started-with-iis/create-a-web-site).
```

**Review by roshnisarangadharan — DISMISSED** [review]
Approved with suggestions.

**Review by acasian-harness — APPROVED** [review]

**Pull request merged**